multiple_listeners/test_mixed.py

```diff
@@ -218,3 +218,65 @@ def test_mixed_https_success(self):
 
         self.check_curl_response(self.make_curl_request("curl-https-true"), fail=False)
         self.check_curl_response(self.make_curl_request("curl-https-false"), fail=True)
+
+
+class single_port_config(TestMixedListeners):
```

Comment:
We use name that start with `Test...`. It is necessary for `-H` test option. And also class name must not contain `_` and start with a capital letter
```suggestion
class TestSinglePortConfig(TestMixedListeners):
```